domain/src/main/java/org/fao/geonet/repository/InspireAtomFeedRepositoryCustomImpl.java

```diff
@@ -61,16 +62,17 @@ public String retrieveDatasetUuidFromIdentifierNs(String datasetIdCode, String d
 
         cbQuery.where(cb.and(datasetIdCodePredicate, datasetIdNsPredicate));
 
-        InspireAtomFeed feed = null;
+        List<InspireAtomFeed> feeds = new ArrayList<>();
 
         try {
-            feed = _entityManager.createQuery(cbQuery).getSingleResult();
+            feeds = _entityManager.createQuery(cbQuery).getResultList();
         } catch (NoResultException nre) {
```

Comment:
This exception is not thrown by the new call to `getResultList` method. The `try-catch` can be removed